lib/reporters/html/index.js

```diff
@@ -122,11 +122,11 @@ function prepareImages(runner) {
     });
 }
 
-module.exports = function htmlReporter(runner) {
+module.exports = function htmlReporter(runner, reportPath) {
     const generateReportPromise = Promise.all([prepareViewData(runner), prepareImages(runner)])
         .spread(view.createHtml)
-        .then(view.save)
-        .then(logPathToHtmlReport)
+        .then((html) => view.save(html, reportPath))
+        .then(logPathToHtmlReport(reportPath))
```

Comment:
@HeeL 

this function will be executed immediately, but we need this function to be executed after `view.save(html, reportPath)`.